### ISSUE TITLE
[VT-59] [vit] inside tmux, line background colours are only present where there's text

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#81]
_Peter Lewis on 2013-02-01T10:38:18Z says:_ 

 In vit 0.7, and only inside a tmux session, when a task is coloured with a background colour, I am only seeing that background colour where there is text. The areas of whitespace between the characters do not get the same background colour. This doesn't look as good as if the whole line's background colour were set.

I'm not sure if this is an issue with vit or with tmux.

For reference, $TERM=screen-256color inside tmux and $TERM=xterm-256color outside tmux. Swapping these has no effect in either case.

Thanks!

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-02-01T10:38:18Z
Modified: 2015-03-19T10:05:10Z
```

**Comment by taskwarrior** [comment]
_steve rader on 2013-03-24T14:30:43Z says:_ 

 The source of the problem here is either 1) tmux's xterm-256color terminal emulation or 2) perl-Curses.  So alas this is not a Vit problem and thus I can not fix it.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-02T00:50:17Z says:_ 

 This is related to VT-108, which was fixed (thanks to Peter Novak) and will be part of VIT 1.3. Peter Lewis, I hope you're still around to enjoy the fix!

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-02T00:51:21Z says:_ 

 Fixed at 09e7acde.

**Comment by taskwarrior** [comment]
_Peter Lewis on 2015-03-19T10:05:10Z says:_ 

 Indeed! All good - thanks.
